src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -48,6 +57,20 @@ export const selectPortfolioAccountBalances = (
   state: ReduxState,
 ): PortfolioAccountBalances['byId'] => state.portfolio.accountBalances.byId
 
+export const selectIsPortfolioLoaded = createSelector(
+  selectAccountSpecifiers,
+  selectPortfolioAssetIds,
+  (accountSpecifiers, portfolioAssetIds) => {
+    if (!accountSpecifiers.length) return false
+    return !size(
```

Comment:
Now that's a great use of lodash helpers here, looks really clean and readable!
To iterate on the declarativeness of this, I'd probably BN it and check for `isZero()`, since someone screening through this without knowing what `lodash/size` does or return

Reply:
difference will always return an array - agree bn is appropriate for values, but this is just data munging stuff